doc/guides/glossary.rst

```diff
@@ -98,7 +98,7 @@ Glossary
 
     Allowed values: ``'MONOCHROME1'``, ``'MONOCHROME2'``, ``'PALETTE COLOR'``,
     ``'RGB'``, ``'YBR_FULL'``, ``'YBR_FULL_422'``, ``'YBR_PARTIAL_420'``,
-    ``'YBR_ICT'``, ``'YBR_RCT'``, however restrictions apply based on
+    ``'YBR_ICT'``, ``'YBR_RCT'``, ``'XYB'``, however restrictions apply based on
```

Comment:
I must admit that I have never heard of XYB or of JPEG XL, for that matter, and had to look it up... Do we actually support these?

Reply:
Nope!